src/Microsoft.OData.Client/DeepInsertSaveResult.cs

```diff
@@ -420,6 +420,7 @@ private void HandleDeepInsertResponseInternal(MaterializerEntry entry, bool isTo
             OperationResponse response = this.CreateOperationResponse(entry, isTopLevelDescriptor, descriptor, parentOperationResponse);
 
             List<Descriptor> relatedDescriptors = this.BulkUpdateGraph.GetRelatedDescriptors(descriptor);
+            int currentRelatedIndex = 0;

```

Comment:
add a comment of what this variable does or is.

Reply:
Will do